diff -Naur SUNSETAddOn_v1.0/sunset_addon/src/Sunset_Driver/Sunset_Evologics/Sunset_Evologics_v1_4/Makefile.am SUNSETAddOn_v1.0_new/sunset_addon/src/Sunset_Driver/Sunset_Evologics/Sunset_Evologics_v1_4/Makefile.am
--- SUNSETAddOn_v1.0/sunset_addon/src/Sunset_Driver/Sunset_Evologics/Sunset_Evologics_v1_4/Makefile.am	2012-05-03 10:42:46.000000000 -0700
+++ SUNSETAddOn_v1.0_new/sunset_addon/src/Sunset_Driver/Sunset_Evologics/Sunset_Evologics_v1_4/Makefile.am	2012-11-02 15:01:25.391799716 -0700
@@ -7,8 +7,12 @@
 				initlib.cc
 
 libSunset_Addon_Evologics_v_one_four_la_CPPFLAGS = @NS_CPPFLAGS@ @NSMIRACLE_CPPFLAGS@
-libSunset_Addon_Evologics_v_one_four_la_LDFLAGS =  @NS_LDFLAGS@ @NSMIRACLE_LDFLAGS@ 
-libSunset_Addon_Evologics_v_one_four_la_LIBADD =   @NS_LIBADD@ @NSMIRACLE_LIBADD@ -lpthread  -L${UW_PATH_LIB_NEW}/libSunset_Core_Driver.so.0.0.0
+libSunset_Addon_Evologics_v_one_four_la_LDFLAGS =  @NS_LDFLAGS@ @NSMIRACLE_LDFLAGS@ -L${SUNSET_PATH}/sunset_lib/lib
+libSunset_Addon_Evologics_v_one_four_la_LIBADD =   @NS_LIBADD@ @NSMIRACLE_LIBADD@ -lpthread -lSunset_Core_Debug -lSunset_Core_Driver \
+									-lSunset_Core_Statistics -lSunset_Core_Real_Scheduler \
+									-lSunset_Core_Utilities -lSunset_Core_Channel_Phy \
+									-lSunset_Core_Connection -lSunset_Core_PktConverter \
+									 -lSunset_Core_Timing
 
 nodist_libSunset_Addon_Evologics_v_one_four_la_SOURCES = initTcl.cc
 BUILT_SOURCES = initTcl.cc
diff -Naur SUNSETAddOn_v1.0/sunset_addon/src/Sunset_Driver/Sunset_Evologics/Sunset_Evologics_v1_6/Makefile.am SUNSETAddOn_v1.0_new/sunset_addon/src/Sunset_Driver/Sunset_Evologics/Sunset_Evologics_v1_6/Makefile.am
--- SUNSETAddOn_v1.0/sunset_addon/src/Sunset_Driver/Sunset_Evologics/Sunset_Evologics_v1_6/Makefile.am	2012-05-03 10:42:46.000000000 -0700
+++ SUNSETAddOn_v1.0_new/sunset_addon/src/Sunset_Driver/Sunset_Evologics/Sunset_Evologics_v1_6/Makefile.am	2012-11-02 15:01:32.807574485 -0700
@@ -7,8 +7,12 @@
 				initlib.cc
 
 libSunset_Addon_Evologics_v_one_six_la_CPPFLAGS = @NS_CPPFLAGS@ @NSMIRACLE_CPPFLAGS@
-libSunset_Addon_Evologics_v_one_six_la_LDFLAGS =  @NS_LDFLAGS@ @NSMIRACLE_LDFLAGS@ 
-libSunset_Addon_Evologics_v_one_six_la_LIBADD =   @NS_LIBADD@ @NSMIRACLE_LIBADD@ -lpthread  -L${UW_PATH_LIB_NEW}/libSunset_Core_Driver.so.0.0.0
+libSunset_Addon_Evologics_v_one_six_la_LDFLAGS =  @NS_LDFLAGS@ @NSMIRACLE_LDFLAGS@ -L${SUNSET_PATH}/sunset_lib/lib
+libSunset_Addon_Evologics_v_one_six_la_LIBADD =   @NS_LIBADD@ @NSMIRACLE_LIBADD@ -lpthread -lSunset_Core_Debug -lSunset_Core_Driver \
+									-lSunset_Core_Statistics -lSunset_Core_Real_Scheduler \
+									-lSunset_Core_Utilities -lSunset_Core_Channel_Phy \
+									-lSunset_Core_Connection -lSunset_Core_PktConverter \
+									 -lSunset_Core_Timing
 
 nodist_libSunset_Addon_Evologics_v_one_six_la_SOURCES = initTcl.cc
 BUILT_SOURCES = initTcl.cc
diff -Naur SUNSETAddOn_v1.0/sunset_addon/src/Sunset_Mac/Sunset_SlottedCsma/Makefile.am SUNSETAddOn_v1.0_new/sunset_addon/src/Sunset_Mac/Sunset_SlottedCsma/Makefile.am
--- SUNSETAddOn_v1.0/sunset_addon/src/Sunset_Mac/Sunset_SlottedCsma/Makefile.am	2012-05-22 02:11:39.000000000 -0700
+++ SUNSETAddOn_v1.0_new/sunset_addon/src/Sunset_Mac/Sunset_SlottedCsma/Makefile.am	2012-11-02 15:01:13.064173841 -0700
@@ -6,8 +6,10 @@
 				initlib.cc
 
 libSunset_Addon_Slotted_Csma_la_CPPFLAGS = @NS_CPPFLAGS@ @NSMIRACLE_CPPFLAGS@
-libSunset_Addon_Slotted_Csma_la_LDFLAGS =  @NS_LDFLAGS@ @NSMIRACLE_LDFLAGS@ 
-libSunset_Addon_Slotted_Csma_la_LIBADD =   @NS_LIBADD@ @NSMIRACLE_LIBADD@  -L${UW_PATH_LIB_NEW}/libSunset_Core_Mac.so.0.0.0
+libSunset_Addon_Slotted_Csma_la_LDFLAGS =  @NS_LDFLAGS@ @NSMIRACLE_LDFLAGS@ -L${SUNSET_PATH}/sunset_lib/lib
+libSunset_Addon_Slotted_Csma_la_LIBADD =   @NS_LIBADD@ @NSMIRACLE_LIBADD@  -lSunset_Core_Debug -lSunset_Core_Statistics \
+									-lSunset_Core_Real_Scheduler -lSunset_Core_Agent  -lSunset_Core_Timing \
+									-lSunset_Core_Utilities -lSunset_Core_PktConverter -lSunset_Core_Mac
 
 nodist_libSunset_Addon_Slotted_Csma_la_SOURCES = initTcl.cc
 BUILT_SOURCES = initTcl.cc
diff -Naur SUNSETAddOn_v1.0/sunset_addon/src/Sunset_Mac/Sunset_Tdma/Makefile.am SUNSETAddOn_v1.0_new/sunset_addon/src/Sunset_Mac/Sunset_Tdma/Makefile.am
--- SUNSETAddOn_v1.0/sunset_addon/src/Sunset_Mac/Sunset_Tdma/Makefile.am	2012-05-22 06:48:49.000000000 -0700
+++ SUNSETAddOn_v1.0_new/sunset_addon/src/Sunset_Mac/Sunset_Tdma/Makefile.am	2012-11-02 15:01:17.264046425 -0700
@@ -5,8 +5,10 @@
 				initlib.cc
 
 libSunset_Addon_Tdma_la_CPPFLAGS = @NS_CPPFLAGS@ @NSMIRACLE_CPPFLAGS@
-libSunset_Addon_Tdma_la_LDFLAGS =  @NS_LDFLAGS@ @NSMIRACLE_LDFLAGS@ 
-libSunset_Addon_Tdma_la_LIBADD =   @NS_LIBADD@ @NSMIRACLE_LIBADD@
+libSunset_Addon_Tdma_la_LDFLAGS =  @NS_LDFLAGS@ @NSMIRACLE_LDFLAGS@ -L${SUNSET_PATH}/sunset_lib/lib
+libSunset_Addon_Tdma_la_LIBADD =   @NS_LIBADD@ @NSMIRACLE_LIBADD@ -lSunset_Core_Debug -lSunset_Core_Statistics \
+						-lSunset_Core_Real_Scheduler  -lSunset_Core_Agent  -lSunset_Core_Timing \
+						-lSunset_Core_Utilities -lSunset_Core_PktConverter  -lSunset_Core_Mac
 
 nodist_libSunset_Addon_Tdma_la_SOURCES = initTcl.cc
 BUILT_SOURCES = initTcl.cc
diff -Naur SUNSETAddOn_v1.0/sunset_addon/src/Sunset_Routing/Sunset_Flooding/Makefile.am SUNSETAddOn_v1.0_new/sunset_addon/src/Sunset_Routing/Sunset_Flooding/Makefile.am
--- SUNSETAddOn_v1.0/sunset_addon/src/Sunset_Routing/Sunset_Flooding/Makefile.am	2012-05-03 10:42:52.000000000 -0700
+++ SUNSETAddOn_v1.0_new/sunset_addon/src/Sunset_Routing/Sunset_Flooding/Makefile.am	2012-11-02 15:01:06.848362305 -0700
@@ -5,8 +5,10 @@
 				initlib.cc
 
 libSunset_Addon_Flooding_la_CPPFLAGS = @NS_CPPFLAGS@ @NSMIRACLE_CPPFLAGS@
-libSunset_Addon_Flooding_la_LDFLAGS =  @NS_LDFLAGS@ @NSMIRACLE_LDFLAGS@ 
-libSunset_Addon_Flooding_la_LIBADD =   @NS_LIBADD@ @NSMIRACLE_LIBADD@   -L${UW_PATH_LIB_NEW}/libSunset_Core_Routing.so.0.0.0
+libSunset_Addon_Flooding_la_LDFLAGS =  @NS_LDFLAGS@ @NSMIRACLE_LDFLAGS@ -L${SUNSET_PATH}/sunset_lib/lib
+libSunset_Addon_Flooding_la_LIBADD =   @NS_LIBADD@ @NSMIRACLE_LIBADD@   -lSunset_Core_Debug -lSunset_Core_Statistics \
+									-lSunset_Core_Utilities -lSunset_Core_PktConverter \
+									-lSunset_Core_Routing -lSunset_Core_Real_Scheduler -lSunset_Core_Timing 
 
 nodist_libSunset_Addon_Flooding_la_SOURCES = initTcl.cc
 BUILT_SOURCES = initTcl.cc
